### PR TITLE
Release/r103

### PR DESCRIPTION
**Pull request by BenFradet**
- [x] remove my registry from spark enrich
- [x] dates (cl, bp, wiki)

### CODE REVIEW

File: 3-enrich/emr-etl-runner/config/config.yml.sample [hunk 1]
```diff
@@ -61,7 +61,7 @@ collectors:
   format: cloudfront # For example: 'clj-tomcat' for the Clojure Collector, 'thrift' for Thrift records, 'tsv/com.amazon.aws.cloudfront/wd_access_log' for Cloudfront access logs or 'ndjson/urbanairship.connect/v1' for UrbanAirship Connect events
 enrich:
   versions:
-    spark_enrich: 1.12.0 # Version of the Spark Enrichment process
+    spark_enrich: 1.13.0 # Version of the Spark Enrichment process
```
Comment (chuwy) [inline 3-enrich/emr-etl-runner/config/config.yml.sample]: Just heads-up that we also have `stream_config.yml.sample` now, but on the other hand `spark_enrich` there is useless and subject for removal in future https://github.com/snowplow/snowplow/issues/3711 (up to you to bump it or not)

File: 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/utils/JsonUtils.scala [hunk 2]
```diff
@@ -193,7 +194,8 @@ object JsonUtils {
    */
   def extractJson(field: String, instance: String): Validation[String, JsonNode] =
     try {
-      Mapper.readTree(instance).success
+      Option(Mapper.readTree(instance))
+        .toSuccess(s"Field [$field]: invalid JSON [$instance] with parsing error: mapping resulted in null")
     } catch {
       case e: Throwable =>
```
Comment (chuwy) [inline 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/utils/JsonUtils.scala]: Should we change it to `NonFatal`?

File: 3-enrich/spark-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.spark/EnrichJobSpec.scala [hunk 3]
```diff
@@ -243,6 +243,16 @@ object EnrichJobSpec {
                 |"uri": "http://iglucentral.com"
               |}
             |}
+          |},
+          |{
+            |"name": "Ben",
```
Comment (chuwy) [inline 3-enrich/spark-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.spark/EnrichJobSpec.scala]: We're not going to delete for production?

Comment (BenFradet) [reply 3-enrich/spark-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.spark/EnrichJobSpec.scala]: it's in the checklist of the pr above to rm it once an iglu central release has gone out

Comment (chuwy) [reply 3-enrich/spark-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.spark/EnrichJobSpec.scala]: Sorry, I just checked if temporary commit exists.

File: 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/utils/JsonUtils.scala (hunk 2 above)
Comment (BenFradet) [reply 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/utils/JsonUtils.scala]: cant hurt :+1:

**Pull request merged**